src/site/content/en/blog/ecommerce-notifications-best-practice/index.md

```diff
@@ -0,0 +1,360 @@
+---
+title: Use best practices for e-commerce notifications
+subhead: Push notifications can help you engage with your customers. Here's how to do them right. 
+authors:
+  - katejeffreys
+date: 2019-07-30
+hero: hero.jpg
+alt: A man looking at his phone at sunset. Photo by @brunoreyna on unsplash.com.
+description: |
+  Learn best practices for implementing push notifications in an e-commerce web app.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - notifications
+  - push
+  - e-commerce
+---
+
+When used effectively, push notifications help your customers to get the most
+out of their online shopping experience. 
+
+Targeted, timely, respectful notifications help people to engage more deeply
+with your site, and build trust in your brand–leading to satisfied customers
+and more repeat shoppers.
+
+With push notifications:
+
+*   Remind a customer about something they need to do–for example, finalize
+    their cart, enter their zipcode, or update their credit card.
+
+*   Tell customers about something new–for example, a special deal, a new
+    product, or an event in their area.
+
+*   Send a customer some information they are expecting, or have
+    requested–for example, you’ve processed or shipped their order, or a
+    product they want to know about has come back in stock.
+
+{% Aside %}
+
+The code samples in this article are drawn from a [sample Glitch app](https://glitch.com/~ecommerce-notifications)
+that we built to accompany this post. You can also see a [working preview](https://ecommerce-notifications.glitch.me/)
+of the app.
+
+{% endAside %}
+
+### Request push notification permissions respectfully
+
+Some sites immediately ask users to permit push notifications. You might see
+this design choice on large social networking and e-commerce sites.
+
+However, customers might be less willing to tolerate this behavior from sites
+they are not invested in. To build trust and retain customers, implement
+notifications in a way that respects the customer and adds to their experience.
+
+{% Aside %}
+
+While the current web platform lets you request notification permissions at
```

Comment:
Not sure 'the web platform' (singular) sounds quite right. Maybe 'While web platforms currently let you request ...'

Also (nit) I think it should be notification _permission_ (not _permissions_): there's only one.